client/state/stats/lists/test/actions.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * External dependencies
+ */
+import sinon from 'sinon';
+import { expect } from 'chai';
+import nock from 'nock';
+
+/**
+ * Internal dependencies
+ */
+import {
+	SITE_STATS_RECEIVE,
+	SITE_STATS_REQUEST,
+	SITE_STATS_REQUEST_FAILURE,
+	SITE_STATS_REQUEST_SUCCESS
+} from 'state/action-types';
+import {
+	receiveSiteStats,
+	requestSiteStats
+} from '../actions';
+
+const siteId = 2916284;
```

Comment:
I could have sworn our JavaScript docs had included comments about using UPPER_CASE to name variable constants, but am failing to find any reference. It's certainly a convention we've adopted. Do you think we should use it here, and separately update the JavaScript docs to include a snippet?